internal/ui/keyboard_js.go

```diff
@@ -0,0 +1,26 @@
+// +build js
+
+package ui
+
+import (
+	"unicode"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+func keypress(e *js.Object) {
+	if runebuffer != nil {
+		if r := rune(e.Get("charCode").Int()); unicode.IsPrint(r) {
+			runebuffer = append(runebuffer, r)
+		}
+	}
+}
+
+func Keyboard() []rune {
```

Comment:
BTW, I feel like there should be a better name than Keyboard, but not sure. Keyboard is confusing with existing APIs like IsKeyPressed. How about InputChars?